INSTALL_LINUX.md

```diff
@@ -129,10 +130,26 @@ git submodule update -i --recursive
 ### Download OpenColorIO-Configs
 
 In the past, OCIO configs were a submodule, though due to the size of the repository, we have chosen instead
-to make a tarball release and let you download it [here](https://github.com/NatronGitHub/OpenColorIO-Configs/archive/Natron-v2.1.tar.gz).
-Place it at the root of Natron repository.
+to make a tarball release and let you download it [here](https://github.com/NatronGitHub/OpenColorIO-Configs/archive/Natron-v3.0.tar.gz).
+
+Create a 'Resources' folder at the root of Natron repository and place it there.
```

Comment:
No: the OpenColorIO-Configs dir has to be in the toplevel source dir.

Reply:
This didn't seems to be the case on my side (I build the sdk and natron on Linux centos 7.6.1810).
I didn't do a "make install", just a make and run natron from the build folder (executing the following from the root of the build folder :  `./App/Natron`

If I put the OpenColorIO-Configs folder at the root of my build folder OR if it is not anywhere, I got a popup at startup : 

```The current OCIO config selected in the preferences is not the default one (blender), would you like to set it to the default config?```

If I click yes, another popup told me it didn't find the files :

```
The blender OCIO config could not be found.
This is probably because you're not using the OpenColorIO-Configs folder 
that should be bundled with your Natron installation.
```
Now If I put the "OpenColorIO-Configs" in a "Resources" folder at the root of my build folder, I get no popup at all. And going in menu Edit|preferences|ColorManagement I see it is already set to "blender" and I can select the other config (that correspond to what is in the Resources/OpenColorIO-Configs folder, if I add a new one in the folder I see it in the menu).

Also in the shell I start natron I see the following trace concerning OpenColor (didn't get those trace otherwise) : 

> Attempt to read an OpenColorIO configuration but the configuration directory "/usr/share/OpenColorIO-Configs" does not exist. 
Attempt to read an OpenColorIO configuration but the configuration directory "/home/yves/vantrix_src/Natron/Natron/App/../share/OpenColorIO-Configs" does not exist.

Finally the reason I found I need that Resources folder was to look at the tree structure of the official Natron portable package. It use the same structure